core/src/main/java/io/fabric8/maven/core/openshift/BuildService.java

```diff
@@ -63,7 +63,7 @@ public Build startBuild(OpenShiftClient client, File dockerTar, String buildName
             }
             if (exp.getCause() instanceof IOException && exp.getCause().getMessage().contains("Stream Closed")) {
                 log.error("Build for %s failed: %s", buildName, exp.getCause().getMessage());
-                log.error("If you are refering to an ImageStream as S2I builder image, please ensure that this ImageStream exists (with 'oc get is')");
+                log.error("If you are refering to an ImageStream as S2I builder image, please ensure that this ImageStream exists (with 'oc get is -n openshift')");
```

Comment:
@jimmidyson printing the users to the right namespace in case they changed the default, or pointing them to check the docker image or the image stream, require some refactoring. I think we can leave as it is and postpone the refactoring (we are going to release the first version shortly).